### PR TITLE
添加redis的认证

### PR DESCRIPTION
**Pull request by lework**

### CODE REVIEW

File: input/redis/inputredis.go [hunk 1]
```diff
@@ -45,6 +47,8 @@ func DefaultInputConfig() InputConfig {
 			},
 		},
 		Host:            "localhost:6379",
+		DB:              0,
+		Password:		 "",
```
Comment (tsaikd) [inline input/redis/inputredis.go:51]: Please run `gofmt`

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: Gogstash
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```
Comment (tsaikd) [inline .github/workflows/build.yml:2]: Why?

Comment (lework) [reply .github/workflows/build.yml:2]: add a manual trigger to execute the github action.

File: input/redis/inputredis.go (hunk 1 above)
Comment (lework) [reply input/redis/inputredis.go]: Sorry, I've amended this one.

File: .github/workflows/build.yml (hunk 2 above)
Comment (tsaikd) [reply .github/workflows/build.yml:2]: When you need to trigger manually?

Comment (lework) [reply .github/workflows/build.yml:2]: test history version.

**Pull request merged**